docker/reqmgr2ms/run.sh

```diff
@@ -40,21 +40,20 @@ if [ -f /etc/secrets/hmac ]; then
     sudo chown $USER.$USER /data/srv/current/auth/$srv/header-auth-key
 fi
 
-# use service configuration files from /etc/secrets if they are present
+### For this specific reqmgr2ms service, we need to replace the default
+# configuration files, by those provided under the proper services_config dir
 cdir=/data/srv/current/config/$srv
-# check if /etc/secrets area contain at least one dbs configuration files
 cfiles="config-monitor.py config-output.py config-transferor.py"
+pushd $cdir
+rm $cfiles
```

Comment:
The logic is incorrect here. If you rm all files it means that your POD will not start the service if you k8s do not provide config files. You only need to remove them ONLY if at least one file is present in the k8s configuration area, and leave all files to allow default behavior.

Reply:
That's a valid use case, let me update it.